plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -91,12 +92,15 @@ private DeltaLakeSchemaSupport() {}
 
     private static final Logger log = Logger.get(DeltaLakeSchemaSupport.class);
 
+    private static final Splitter SPLITTER = Splitter.on(',').trimResults().omitEmptyStrings().trimResults();
+
     public static final String APPEND_ONLY_CONFIGURATION_KEY = "delta.appendOnly";
     public static final String COLUMN_MAPPING_MODE_CONFIGURATION_KEY = "delta.columnMapping.mode";
     public static final String COLUMN_MAPPING_PHYSICAL_NAME_CONFIGURATION_KEY = "delta.columnMapping.physicalName";
     public static final String MAX_COLUMN_ID_CONFIGURATION_KEY = "delta.columnMapping.maxColumnId";
     public static final String ISOLATION_LEVEL_CONFIGURATION_KEY = "delta.isolationLevel";
     private static final String DELETION_VECTORS_CONFIGURATION_KEY = "delta.enableDeletionVectors";
+    private static final String UNIVERSAL_FORMAT_CONFIGURATION_KEY = "delta.universalFormat.enabledFormats";
```

Comment:
Is this documented at https://github.com/delta-io/delta/blob/master/PROTOCOL.md ?

if not, let's document where this comes from

Reply:
It comes from https://github.com/delta-io/delta/blob/master/docs/source/delta-uniform.md. I will add a code comment in the next push.